frontend/public/components/factory/modal.tsx

```diff
@@ -90,7 +90,7 @@ export const ModalTitle: React.SFC<ModalTitleProps> = ({
 export const ModalBody: React.SFC<ModalBodyProps> = ({ children }) => (
   <div className="modal-body">
     <div className="modal-body-content">
-      <div className="modal-body-inner-shadow-covers">{children}</div>
+      <div className="modal-body-inner-shadow-covers pf-c-content">{children}</div>
```

Comment:
`pf-c-content` is the classname resulting from [`<TextContent>`](https://www.patternfly.org/v4/components/text#textcontent), so I think it'd be better to use [`<TextContent>`](https://www.patternfly.org/v4/components/text#textcontent) here and the other places `pf-c-content` is needed instead of hard-coding the classname.  Note you can pass additional classnames to [`<TextContent>`], but probably want to add it as a standalone node in `markdown-view.tsx`.